package/doc/sphinx/source/documentation_pages/references.rst

```diff
@@ -65,170 +51,182 @@ Citations for included algorithms and modules
 
 If you use the RMSD calculation (:mod:`MDAnalysis.analysis.rms`) or alignment
 code (:mod:`MDAnalysis.analysis.align`) that uses the
-:mod:`~MDAnalysis.core.qcprot` module please also cite [Theobald2005b]_ and
-[Liu2010b]_.
+:mod:`~MDAnalysis.core.qcprot` module please also cite [Theobald2005b]_
+and :cite:p:`a-Liu2010`.
+
+.. bibliography::
+   :filter: False
+   :style: MDA
+   :keyprefix: a-
+   :labelprefix: ᵃ
+
+   Liu2010
 
 .. [Theobald2005b] Douglas L. Theobald. Rapid calculation of RMSD using a
    quaternion-based characteristic polynomial. *Acta Crystallographica A*
    **61** (2005), 478-480.
 
-.. [Liu2010b] Pu Liu, Dmitris K. Agrafiotis, and Douglas L. Theobald. Fast
-   determination of the optimal rotational matrix for macromolecular
-   superpositions. *J. Comput. Chem.* **31** (2010), 1561–1563.
-
 If you use the helix analysis algorithm HELANAL_ in
-:mod:`MDAnalysis.analysis.helanal` please cite [Bansal2000b]_.
+:mod:`MDAnalysis.analysis.helix_analysis` please cite :cite:p:`a-Bansal2000`.
+
+.. bibliography::
+   :filter: False
+   :style: MDA
+   :keyprefix: a-
+   :labelprefix: ᵃ
 
-.. [Bansal2000b] Bansal M, Kumar S, Velavan R. HELANAL — A program to
-   characterise helix geometry in proteins. *J. Biomol. Struct. Dyn.* **17**
-   (2000), 811–819
+   Bansal2000
 
 .. _HELANAL: http://www.ccrnp.ncifcrf.gov/users/kumarsan/HELANAL/helanal.html
```

Comment:
Although I could have added this URL to the bibtex reference, I left it just in case this is preferred. Otherwise, I can pass the link to the reference and show the URL within the reference.

Reply:
Leave it separate, the link is to the code, not the paper.